.appveyor.yml

```diff
@@ -40,9 +44,13 @@ cache:
   - target
 
 build_script:
-  - cmd: cargo build --target %TARGET%
-  - cmd: cargo build --release --target %TARGET%
+  - cargo build --all --target %TARGET%
+  - cargo build --all --release --target %TARGET%
 
 test_script:
-  - cmd: cargo test --target %TARGET%
-  - cmd: cargo test --release --target %TARGET%
+  - cargo test --all --target %TARGET%
+  - cargo test --all --release --target %TARGET%
+
+on_failure:
+  - tree target /F /A
```

Comment:
/F /A?

Reply:
Sorry the tree command was only helpfull when I was trying to figure out the windows build layout.